integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/viewmodels/ConvertViewViewModel.kt

```diff
@@ -61,7 +62,7 @@ class ConvertViewViewModel @Inject constructor(
     var minAllowedSwapAmount: String = "2"
 
     var maxForDashWalletAmount: String = "0"
-    val onContinueEvent = SingleLiveEvent<Pair<Boolean, Pair<Fiat?, Coin?>?>>()
+    val onContinueEvent = SingleLiveEvent<SwapRequest>()
```

Comment:
This `Pair<Boolean, Pair<Fiat?, Coin?>?>` construction is ugly and hard to understand. Might be a good place to use a specialized model class.